File: frontend/src/app/DTO/appointment-review-dto.ts
export class AppointmentReviewDto {
    bloodBankId?: number;
    customerId?: number;

    public constructor(bbId?: number, cId?: number) {
        if (bbId || cId) {
            this.bloodBankId = bbId;
            this.customerId = cId;
        }
    }
}

File: g3malt/src/main/java/isaapp/g3malt/dto/AppointmentReviewDto.java
package isaapp.g3malt.dto;

public class AppointmentReviewDto {
	
	private Integer bloodBankId;
    private Integer customerId;
    
    public AppointmentReviewDto() {}
    
	public AppointmentReviewDto(Integer bloodBankId, Integer customerId) {
		super();
		this.bloodBankId = bloodBankId;
		this.customerId = customerId;
	}
	
	public Integer getBloodBankId() {
		return bloodBankId;
	}
	public void setBloodBankId(Integer bloodBankId) {
		this.bloodBankId = bloodBankId;
	}
	public Integer getCustomerId() {
		return customerId;
	}
	public void setCustomerId(Integer customerId) {
		this.customerId = customerId;
	}
}
